src/main/java/frc/lib/io/detectionML/detection_ML_IO.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2025 Windham Windup
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program. If
+ * not, see <https://www.gnu.org/licenses/>.
+ */
+
+package frc.lib.io.detectionML;
+
+import org.littletonrobotics.junction.AutoLog;
+
+/**
+ * Standardized interface for ML-IO used in FRC.
+ */
+@AutoLog
+public interface detection_ML_IO {
+    /*
```

Comment:
Instead of returnXXX(), move all of those values into the autologged field, except for cameraString. and add updateInputs(XXX XXXX) at the bottom of this file

Reply:
I would use the distance sensor as a template for structure https://github.com/WHS-FRC-3467/W8-Library/blob/dev/src/main/java/frc/lib/io/distancesensor/DistanceSensorIO.java